packages/devtools_app/lib/src/screens/memory/panes/allocation_tracing/allocation_profile_class_table.dart

```diff
@@ -0,0 +1,147 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import '../../../../primitives/utils.dart';
+import '../../../../shared/common_widgets.dart';
+import '../../../../shared/table.dart';
+import '../../../../shared/table_data.dart';
+import '../../../../shared/utils.dart';
+import 'allocation_profile_tracing_view_controller.dart';
+
+/// The default width for columns containing *mostly* numeric data (e.g.,
+/// instances, memory).
+const _defaultNumberFieldWidth = 80.0;
+
+class _TraceCheckBoxColumn extends ColumnData<TracedClass>
+    implements ColumnRenderer<TracedClass> {
+  _TraceCheckBoxColumn()
+      : super(
+          'Trace',
+          titleTooltip:
+              'Enable or disable allocation tracing for a specific type',
+          fixedWidthPx: scaleByFontFactor(55.0),
+          alignment: ColumnAlignment.left,
+        );
+
+  @override
+  bool get supportsSorting => false;
+
+  @override
+  Widget build(
+    BuildContext context,
+    TracedClass item, {
+    bool isRowSelected = false,
+    VoidCallback? onPressed,
+  }) {
+    final controller =
+        Provider.of<AllocationProfileTracingViewController>(context);
+    return Checkbox(
+      value: item.traceAllocations,
+      onChanged: (value) async {
+        await controller.setAllocationTracingForClass(item.cls, value!);
+      },
+    );
+  }
+
+  @override
+  bool? getValue(TracedClass _) {
+    return null;
+  }
+
+  @override
+  int compare(TracedClass a, TracedClass b) {
+    return a.traceAllocations.boolCompare(b.traceAllocations);
+  }
+}
+
+class _ClassNameColumn extends ColumnData<TracedClass> {
+  _ClassNameColumn() : super.wide('Class');
+
+  @override
+  String? getValue(TracedClass stats) => stats.cls.name;
+
+  @override
+  bool get supportsSorting => true;
+}
+
+class _InstancesColumn extends ColumnData<TracedClass> {
+  _InstancesColumn()
+      : super(
+          'Instances',
+          fixedWidthPx: scaleByFontFactor(_defaultNumberFieldWidth),
+        );
+
+  @override
+  int getValue(TracedClass dataObject) {
+    return dataObject.instances;
+  }
+
+  @override
+  bool get numeric => true;
+}
+
+class AllocationTracingTable extends StatefulWidget {
+  const AllocationTracingTable({required this.controller});
+
+  final AllocationProfileTracingViewController controller;
+
+  @override
+  State<AllocationTracingTable> createState() => _AllocationTracingTableState();
+}
+
+class _AllocationTracingTableState extends State<AllocationTracingTable> {
+  late SortDirection sortDirection;
```

Comment:
I traced 'Text', collected 7 instances, then I unchecked it. 
I see number 7 in 'Instances' and I expect that the Text is not tracked any more, but the collected information preserved. 
But, when I select this row, I do not see the collected information and, when I order by number of instances, Text does not get to the top or bottom:
<img width="1043" alt="Screen Shot 2022-08-12 at 9 42 18 AM" src="https://user-images.githubusercontent.com/12115586/184405237-091bb829-66d4-4c6c-b97e-5f441378e3e2.png">

Reply:
This should be fixed now.